### DISCUSSION TITLE
Installation issue

### DISCUSSION THREAD

**Discussion by haladslader** [ScoopInstaller/Scoop#5429] [Help and Questions]
"Scoop requires 'C:\Windows\System32\Robocopy.exe' to work. Please make sure 'C:\Windows\System32' is in your PATH.
Abort."
This is what i get after running the command to install, probably an easy solution but im confused.

**Comment by rashil2000** [comment]
See https://github.com/ScoopInstaller/Scoop/discussions/5364#discussioncomment-4826257
